UseMultiArch.cmake

```diff
@@ -20,7 +20,7 @@ if ("${CMAKE_SYSTEM_NAME}" MATCHES "Linux" AND
   if (EXISTS "/etc/debian_version")
 	set (_libdir_def "lib/${CMAKE_LIBRARY_ARCHITECTURE}")
 	set (_libdir_noarch "lib")
-  else (EXISTS "/etc/debian_version")
+  elseif (EXISTS "/etc/debian_version")
```

Comment:
this is incorrect as it will again test for "/etc/debian_version"
cmake if syntax is:
>if(expression)
>    ...
>   elseif(expression2)
>     ...
>   else(expression)
>     ...
>   endif(expression)